### PR TITLE
WhiteSource Local config file

### PR DESCRIPTION
**Pull request by mukul-seagate11**
WhiteSource Local config file will be created to scan & detect vulnerabilities based on local component configurations/Artifacts/code extensions etc.

### CODE REVIEW

File: whitesource-components.config [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+checkPolicies=false
+forceCheckAllDependencies=false
+forceUpdate=false
+forceUpdate.failBuildOnPolicyViolation=false
+updateInventory=true		  
+
+###########################################################################################
+# Includes/Excludes Glob patterns - Please use only one exclude line and one include line #
+###########################################################################################
+includes=**/*.c **/*.cc **/*.cp **/*.cpp **/*.cxx **/*.c++ **/*.h **/*.hpp **/*.hxx
```
Comment (indrajitzagade) [inline whitesource-components.config:29]: Most of HA code is in python, we need to update this include python files I guess.

Comment (mukul-seagate11) [reply whitesource-components.config]: done, included .py as well

**Comment by ajaysrivas** [comment]
@mukul-seagate11 please fix the DCO and add relevant reviewers.

**Comment by mukul-seagate11** [comment]
> @mukul-seagate11 please fix the DCO and add relevant reviewers.

Done

**Comment by ajaysrivas** [comment]
@mukul-seagate11 Please check why whitesource security check has failed to execute.

**Comment by mukul-seagate11** [comment]
> @mukul-seagate11 Please check why whitesource security check has failed to execute.

WhiteSource check is searching for local config file so this PR has tp be merged to get triggered

**Review by ajaysrivas — APPROVED** [review]

**Pull request merged**